packages/api/src/cli/commands/install.ts

```diff
@@ -89,7 +89,7 @@ cmd
 
           try {
             return Storage.isIdentifierValid(value, true);
-          } catch (err) {
+          } catch (err: any) {
```

Comment:
I'm gonna swap these out with the [useUnknownInCatchVariables](https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables) TSConfig option.